prehook/richcontainer/richcontainer_hook.go

```diff
@@ -123,3 +130,40 @@ func getRichModeLauncher(spec *specs.Spec) (RichContainerLauncher, error) {
 
 	return launcher, nil
 }
+
+const (
```

Comment:
We do not recommend adding const variables in the middle of file, please add this at the head of it . Thanks a lot  @wangforthinker